File: CinemaTicket.kt/src/domain/models/MovieRepository.kt
package domain.models

class MovieRepository {
    companion object {
        val movies: Array<Movie> = arrayOf(
            Movie("title1", true),
            Movie("title2", true),
            Movie("title3", true),
            Movie("title4", true),
            Movie("title5", true),
            Movie("title6", true)
        )
    }

    fun get(id :Int ) :Movie{
        return movies[id];
    }
}

File: CinemaTicket/src/domain/repositories/movieRepository.ts
import { Movie } from '../models/movie';

export default class MovieRepository {
  private movies: Movie[] = [
    new Movie('title1', true),
    new Movie('title2', false),
    new Movie('title3', true),
    new Movie('title4', false),
    new Movie('title5', true),
  ];

  get(id: number): Movie {
    return this.movies[id];
  }
}
